extensions/metrics/src/__tests__/acceptance/metrics-push.acceptance.ts

```diff
@@ -0,0 +1,74 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/rest-explorer
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {RestApplication, RestServerConfig} from '@loopback/rest';
+import {givenHttpServerConfig, supertest} from '@loopback/testlab';
+import * as os from 'os';
+import pEvent from 'p-event';
+import * as path from 'path';
+import {promisify} from 'util';
+import {MetricsComponent} from '../..';
+import {MetricsBindings} from '../../keys';
+import {MetricsOptions} from '../../types';
+
+const runShell = require('@loopback/build/bin/utils').runShell;
+
+// Only run the test on Travis with Linux
+const verb =
+  process.env.TRAVIS && os.platform() === 'linux' ? describe : describe.skip;
+verb('Metrics (with push gateway)', function() {
```

Comment:
Please use `skipIf` or `skipOnTravis` from `@loopback/testlab`.

Example usage:

https://github.com/strongloop/loopback-next/blob/6ad0bb59c71ea2356cb951da786bdbff246b47e7/packages/repository-tests/src/crud/freeform-properties.suite.ts#L26-L30

Reply:
> `skipIf(describe, '...', () => {});` does not compile.

That's a known limitation of the current version and/or TypeScript.

Did you try the slightly-longer form I shown in my comment?

```ts
skipIf<[(this: Suite) => void], void>( 
```